### ISSUE TITLE
Contao 4.13 Support

### ISSUE THREAD

**Issue by vossmedien** [menatwork/syncCto#317]
Hey :-)
thank you for this extension!
is it planned to support the current 4.13 contao version?

thanks!

Christian

**Comment by mammut47** [comment]
That would be just as interesting for me
Manfred

**Comment by stefanheimes** [comment]
Hi, yes we will support it. We are on it and check what we have to do. But there is still a problem with the cca/dc_general. The DC-General isn't ready for Contao 4.13 or PHP 8.*.  We have to speak with the CCA Group and find a solution for this.

**Comment by nlc21** [comment]
Hallo Stefan,
gibt es eine Möglichkeit, dies zu beschleunigen - z.B. durch finanzielle Unterstützung?
Gruß,
Felix Pfeiffer

**Comment by mammut47** [comment]
Ihr habt auf eurer Seite angegeben, dass die Unterstützung für 4.13 auf Juni 2022 geplant ist. Seid ihr da schon weiter gekommen? Ich wäre auch stark daran interessiert, dass synccto auf 4.13 läuft.
Gruß
Manfred Gipp

**Comment by stefanheimes** [comment]
He Manfred, 

ja sind wir. Leider sind wir nicht so schnell voran gekommen wie geplant. Wir sind aber beim letzten Baustein angekommen, neben Synccto haben wir auch noch die CCA Sachen für Contao 4.13 und PHP 8 fertig gemacht und sind gerade am letzten Projekt dran, dem DCG. Wenn dieser durch ist, sollten wir alles haben und eine stabile Version zu erlauben. 

Im Moment wäre es möglich die Dev Versionen zu installieren, was ich von einige schon mitbekommen habe, dass Sie dies machen und SyncCto dort relative gut läuft, bis auf die Tatsache das es halt Dev Versionen zum Teil sind.  

Wir sind aber aktiv dran.

**Comment by zonky2** [comment]
Läuft mit Contao 4.13 und PHP 8.1 prima

**Comment by zonky2** [comment]
wäre eher Frage für Forum - dort bitte auch deine composer.json posten...

**Comment by con2net** [comment]
Danke für die schnelle Antwort! Ich bin leider nicht so fit in diesen Dingen aber ich habe eine ganz nackte/frische Contao Installation ohne weitere Erweiterungen. Wenn ich in die composer.json der syncCto schaue, dann steht da ja, dass die contao-community-alliance/dc-general erforderlich ist und in dieser wiederum steht doch, dass nur PHP 7.4er Versionen erlaubt sind oder? 
Deswegen bittest Du, dass ich mich damit an das community.contao.org-Forum wende, ja? Welches spezifische Forum wäre am geeignetsten?

**Comment by zonky2** [comment]
Bereich wäre https://community.contao.org/de/forumdisplay.php?139-syncCto

Weil das hier ein Ticketsystem für Bugs und Features ist... allgemeine Fragen sind im Forum besser aufgehoben, da mehr leute was von der Antwort haben ;-)

**Comment by con2net** [comment]
Haham oke, danke, ich dachte das wäre ein Bug und genau deshalb war ich hier ;-). Oke, ich poste das noch mal in Deinem Link. Vielen Dank dafür!

**Comment by zonky2** [comment]
es ist nicht immer ein Bug, wenn was nicht wie gedacht funktioniert ... ;-)

**Comment by con2net** [comment]
Ja, sorry, das stimmt natürlich. Ich sehe jetzt auch, dass dort genau das Problem schon beschrieben ist und scheinbar auch eine Abhilfe dargestellt ist. Danke fürs raufschubsen.

**Comment by mammut47** [comment]
Leider lässt sich auch die dev Version bei mir nicht installieren
Contao 4.13 und PHP 8.1
Meldungen:
```
Resolving dependencies using Composer Cloud v3.6.0-1-g9d9036c
[6.8MiB/0.18s] Loading composer repositories with package information
[77.3MiB/8.43s] Updating dependencies
[175.9MiB/9.22s] Your requirements could not be resolved to an installable set of packages.
[175.9MiB/9.22s] 
  Problem 1
    - contao-community-alliance/dc-general[2.1.3, ..., 2.1.9] require php ^7.1 -> your php version (8.1.19.0) does not satisfy that requirement.
    - contao-community-alliance/dc-general[2.2.0, ..., 2.2.3] require php ^7.4 -> your php version (8.1.19.0) does not satisfy that requirement.
    - menatwork/synccto dev-master requires contao-community-alliance/dc-general ^2.1.3 -> satisfiable by contao-community-alliance/dc-general[2.1.3, ..., 2.2.3].
    - Root composer.json requires menatwork/synccto dev-master -> satisfiable by menatwork/synccto[dev-master].
[175.9MiB/9.23s] <warning>Running update with --no-dev does not mean require-dev is ignored, it just means the packages will not be installed. If dev requirements are blocking the update you have to resolve those problems.</warning>
[44.3MiB/9.31s] Memory usage: 44.32MB (peak: 201.33MB), time: 9.31s.
[44.3MiB/9.31s] Finished Composer Cloud resolving.
```
Hier wird immer noch php 7.4 verlangt.
Was läuft da noch falsch?

**Comment by zonky2** [comment]
https://community.contao.org/de/showthread.php?84639-syncCto-f%C3%BCr-Contao-4-13-und-PHP-8&p=569365&viewfull=1#post569365
